lib/redis_cluster/client.rb

```diff
@@ -240,14 +240,13 @@ def refresh_startup_nodes
     # attempt is raised to the user.
     def retry_intermittent_loop
       last_error = nil
-
       for attempt in 0..(@retry_count) do
         begin
           yield(attempt)
 
           # Fall through on any success.
           return
-        rescue Errno::EACCES, Redis::TimeoutError, Redis::CannotConnectError => e
```

Comment:
Removed this error here, as a CannotConnectError is more likely to require a refresh, instead of just a retry.